torch/csrc/jit/codegen/cuda/interface.cpp

```diff
@@ -24,6 +24,103 @@ namespace cuda {
 
 static std::atomic<bool> cuda_fusion_guard_mode{true};
 
+// There are 3 sources of information on whether to enable nvfuser:
+// 1. assigned value from setEnabled() - takes precendence if it has been set
+// 2. value from environment variable - only used if setEnabled() is unset
+// 3. default value - used if both 1 and 2 are unset.
```

Comment:
another nitpick: `default` value would be a little bit tricky since once we started using `torch.jit.fuser` context manager, we can't return to a state where we'll be using `default` any more, since we explicitly called `setEnabled` when exiting.
https://github.com/pytorch/pytorch/blob/master/torch/jit/_fuser.py#L63

But practically we are still respecting the effective flag. So don't see that as a blocking issue